modules/software-extensibility/pages/extensions-sdk.adoc

```diff
@@ -0,0 +1,28 @@
+= Extensions Source Development Kits(SDK)
+:description: Several Source Development Kits are provided so that you can extend your automation project
+
+{description}
+
+
+[.card-section]
+== The provided SDKs
+
+[.card.card-index]
+--
+xref:connector-archetype.adoc[[.card-title]#Connector Maven Archetype# [.card-body.card-content-overflow]#pass:q[Create custom Connectors]#]
+--
+
+[.card.card-index]
+--
+xref:actor-filter-archetype.adoc[[.card-title]#Actor Filter Maven Archetype# [.card-body.card-content-overflow]#pass:q[Create custom Actor Filters]#]
+--
+
+[.card.card-index]
+--
+xref:rest-api-extension-archetype.adoc[[.card-title]#REST API Extension Maven Archetype# [.card-body.card-content-overflow]#pass:q[Create REST API Extensions]#]
+--
+
+[.card.card-index]
+--
+xref:customize-living-application-theme.adoc[[.card-title]#Theme Maven Archetype# [.card-body.card-content-overflow]#pass:q[Create custom Themes]#]
```

Comment:
You have to reference the module since this page is in an other module.
Please use the preview to check your contribution, you would have notified it quickly.

```suggestion
xref:applications:customize-living-application-theme.adoc[[.card-title]#Theme Maven Archetype# [.card-body.card-content-overflow]#pass:q[Create custom Themes]#]
```